### ISSUE TITLE
Testing on Safari for PC 

### ISSUE THREAD

**Issue by kyoukhana** [01org/appframework#41]
In the latest version of safari for Windows you get the following error WebKit2WebProcess has stopped working.  This is created by inserting the following code.  

Copy the entire kitchen sink code then stip everything between the body tags and insert this code

<header id="header">
    <a href="#"><img id="lgo" src="images/misc/placer.png" ></a>
</header>

The code will then crash the browser
remove this line  

<img id="lgo" src="images/misc/placer.png" >

And it works.  

No Errors in Google Chrome 

Basically the img tag is causing a issue
